### ISSUE TITLE
Can't build Ant Design 3.5.4 with uglify-js

### ISSUE THREAD

**Issue by whtsky** [ant-design/ant-design#10667]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Version
3.5.4

### Environment
N/A

### Reproduction link
[https://github.com/whtsky/antd-build-error](https://github.com/whtsky/antd-build-error)

### Steps to reproduce
yarn build

### What is expected?
build success

### What is actually happening?

```
ERROR in Encountered an error while minifying bundle.js:
SyntaxError: In strict mode code, functions can only be declared at top level or immediately within another function.
  40098 |
  40099 | if (__DEV__) {
> 40100 |   function printWarning(format, args) {
        |  ^
  40101 |     var len = arguments.length;
  40102 |     args = new Array(len > 2 ? len - 2 : 0);
  40103 |     for (var key = 2; key < len; key  ) {
```

---
Related: https://github.com/BerkeleyTrue/warning/issues/25

downgrade `warning` to 3.x should solve this

**Comment by AuthurWang** [comment]
v3.5.4 used warning 4.0.0 published 5/28(https://github.com/BerkeleyTrue/warning), this package has an error in strict mode, but has been repaired(https://github.com/BerkeleyTrue/warning/commit/677dcfab1a23883aaaf73ec6e3b9f95e8b656d03#diff-3738be060fbf4418ef8ad8c72dce285d).

maybe antd need re-published version >3.5.4!

**Comment by whtsky** [comment]
@AuthurWang but warning didn't release a new version.
(ehh.. and why did you @ me?)

**Comment by yoyo837** [comment]
`warning`  released V4.0.1 now.
